packages/oauth/src/index.ts

```diff
@@ -291,10 +320,10 @@ export class InstallProvider {
 
     try {
       if (req.url !== undefined) {
-        parsedUrl = parseUrl(req.url, true);
-        code = parsedUrl.query.code as string;
-        state = parsedUrl.query.state as string;
-        if (state === undefined || state === '' || code === undefined) {
```

Comment:
Wondering whether this was written this way to be a bit more explicit for people unfamiliar with the JS type coercion.

Reply:
I rewrote this part as use of `parseUrl` is deprecated and that triggered an eslint rule! So I did it the 'approved' newer-node-version way of creating a `new URL`.